### DISCUSSION TITLE
student can‘t free to use copilot,need some help!

### DISCUSSION THREAD

**Discussion by hourong88** [community/community#37609] [Copilot Conversations]
This year copilot began to charge, I bought. Then I found out that it was free for students, so I certified my student status, and then now it expired and I still can't use it for free, so I don't know what's going on.

<img width="831" alt="image" src="https://user-images.githubusercontent.com/35595174/198523981-cede8886-3511-442e-82f0-77f3d49c6ff3.png">

click the enable button,then jump to here
<img width="684" alt="image" src="https://user-images.githubusercontent.com/35595174/198524101-81aac8bb-4a1e-447c-a474-749652bbbdea.png">

<img width="925" alt="image" src="https://user-images.githubusercontent.com/35595174/198524528-17d2745a-9e5e-4d2f-9f06-9f96e6b46bf2.png">

so i dont know how to going on .

**Comment by iamcjwilliams** [comment]
Sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted
